rust-toolchain.toml

```diff
@@ -1,6 +1,6 @@
 [toolchain]
 profile = "default"
-channel = "1.78.0"
```

Comment:
why?

Reply:
Since I don't have a windows machine to debug with, and the cause of the failure isn't clear (although I agree, it looks like it's trying to dynamically load something and failing), I'm trying to see if changing runners / rustc versions or removing the cached artifacts is helping. It's not, so far. Next step will be trying to see if I can get my hands on a windows VM to reproduce the issue.

Reply:
ah ok makes sense with the commit names 😅